### ISSUE TITLE
0.9.3 webpack error

### ISSUE THREAD

**Issue by davidspiess** [jurassix/react-immutable-render-mixin#15]
I updated to the newest version and get the following error trying to import it

```js
import ImmutableRenderMixin from 'react-immutable-render-mixin';
```

`Module not found: Error: Cannot resolve module 'react-immutable-render-mixin'``

0.9.2 works fine though.

**Comment by crooy** [comment]
we have this issue as well

**Comment by jurassix** [comment]
Ok I'll fix this asap

**Comment by jurassix** [comment]
This should be fixed in v0.9.4

**Comment by davidspiess** [comment]
Thanks for the quick fix!! Everything back to normal :)
